openstack/auth_env.go

```diff
@@ -22,6 +39,16 @@ func AuthOptionsFromEnv() (gophercloud.AuthOptions, error) {
 	domainID := os.Getenv("OS_DOMAIN_ID")
 	domainName := os.Getenv("OS_DOMAIN_NAME")
 
+	// If OS_PROJECT_ID is set, overwrite tenantID with the value.
+	if v := os.Getenv("OS_PROJECT_ID"); v != "" {
```

Comment:
Yeap, you're right that `func AuthOptionsFromEnv` is the only place where both `OS_TENANT_*` and `OS_PROJECT_*` exist next to each other.

So I agree to keep the code as it is.